file_upload_handler/src/server.js

```diff
@@ -0,0 +1,59 @@
+import express from "express";
+import multer from "multer";
+
+const PORT = process.env.PORT || 8000
+const app = express();
+
+const storage = multer.diskStorage({
+  destination: function (req, file, cb) {
+    cb(null, "./uploads");
+  },
+  filename: function (req, file, cb) {
+    const uniquepreffix = Date.now() + "-" + Math.round(Math.random() * 1e9);
+    cb(null, uniquepreffix + "-" + file.originalname);
+  },
+});
+
+function fileFilter(req, file, cb) {
```

Comment:
Documentation required here,
a small description of what the function does, the input parameters and the return value if any